rest/jersey2/src/main/java/org/seedstack/seed/rest/jersey2/internal/Jersey2Plugin.java

```diff
@@ -27,10 +27,7 @@
 import org.seedstack.seed.rest.RestConfig;
 import org.seedstack.seed.rest.internal.RestPlugin;
 import org.seedstack.seed.rest.spi.RestProvider;
-import org.seedstack.seed.web.spi.FilterDefinition;
-import org.seedstack.seed.web.spi.ListenerDefinition;
-import org.seedstack.seed.web.spi.ServletDefinition;
-import org.seedstack.seed.web.spi.WebProvider;
+import org.seedstack.seed.web.spi.*;
```

Comment:
Prefer explicit imports instead of star imports

Reply:
Agree!